auth-manifest/README.md

```diff
@@ -0,0 +1,33 @@
+| Manufacturer ECC Public Key | 96 | ECC P-384 public key used to verify the IMC Signature. <br> **X-Coordinate:** Public Key X-Coordinate (48 bytes) <br> **Y-Coordinate:** Public Key Y-Coordinate (48 bytes) |
+| Manufacturer LMS Public Key | 48 | LMS public key used to verify the IMC Signature. <br> **tree_type:** LMS Algorithm Type (4 bytes) <br> **otstype:** LMS Ots Algorithm Type (4 bytes) <br> **id:**  (16 bytes) <br> **digest:**  (24 bytes) |
+| Manufacturer ECC Signature | 96 | Manufacturer ECDSA P-384 signature of the Version, Flags, Manufacturer ECC and LMS public keys, hashed using SHA2-384. <br> **R-Coordinate:** Random Point (48 bytes) <br> **S-Coordinate:** Proof (48 bytes) |
+| Manufacturer LMS Signature | 1620 | Manufacturer LMS signature of the Version, Flags, Manufacturer ECC and LMS public keys, hashed using SHA2-384. <br> **q:** Leaf of the Merkle tree where the OTS public key appears (4 bytes) <br> **ots:** Lmots Signature (1252 bytes) <br> **tree_type:** Lms Algorithm Type (4 bytes) <br> **tree_path:** Path through the tree from the leaf associated with the LM-OTS signature to the root. (360 bytes) |
+| Owner ECC Public Key | 96 | ECC P-384 public key used to verify the IMC Signature. <br> **X-Coordinate:** Public Key X-Coordinate (48 bytes) <br> **Y-Coordinate:** Public Key Y-Coordinate (48 bytes) |
+| Owner LMS Public Key | 48 | LMS public key used to verify the IMC Signature. <br> **tree_type:** LMS Algorithm Type (4 bytes) <br> **otstype:** LMS Ots Algorithm Type (4 bytes) <br> **id:**  (16 bytes) <br> **digest:**  (24 bytes) |
+| Owner ECC Signature | 96 | Owner ECDSA P-384 signature of the Version, Flags, Owner ECC and LMS public keys, hashed using SHA2-384. <br> **R-Coordinate:** Random Point (48 bytes) <br> **S-Coordinate:** Proof (48 bytes) |
+| Owner LMS Signature | 1620 | Owner LMS signature of the Version, Flags, Owner ECC and LMS public keys, hashed using SHA2-384. <br> **q:** Leaf of the Merkle tree where the OTS public key appears (4 bytes) <br> **ots:** Lmots Signature (1252 bytes) <br> **tree_type:** Lms Algorithm Type (4 bytes) <br> **tree_path:** Path through the tree from the leaf associated with the LM-OTS signature to the root. (360 bytes) |
+
+- ### **Image Metadata Collection**
+The Image Metadata Collection (IMC) is a collection of Image Metadata entries (IME). Each IME has a hash that matches a SOC images. The manifest manufacturer and owner private keys sign the IMC. The Preamble holds the IMC signatures. The manifest IMC manufacturer signatures are optional and are validated only if the FLAGS field Bit 0 = 1. Up to sixteen image hashes will be supported.
+
+| Field | Size (bytes) | Description|
+|-------|--------|------------|
+| Revision | 4 | Version of the IMC structure |
+| Reserved | 4 | Reserved |
+| Image Metadata Entry (IME) Count | 4 | Number of IME(s) in the IMC |
+| Image Metadata Entry (N) | Variable | List of Image Metadata Entry structures |
```

Comment:
Is the size of each entry fixed and "variable" here refers to the fact the entry count just varies? Would it make sense to include the "Image Metadata Entry structure" format in the readme?

Reply:
That's correct. Added the IME format in the README.